crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -782,6 +783,183 @@ async fn test_convert_big_event() {
     assert!(converter.convert(&big_event_message).await.is_empty());
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_convert_big_measurement() {
```

Comment:
The test cases contain a lot of shared code to create an `http_proxy`, a `converter ` then a `mapper`.

=> I would consider some test helpers.

Reply:
Moved the creation of the converter to another helper function.